filters-share/extended_css_INJECTION.txt

```diff
@@ -1,2 +1,3 @@
+www.instiz.net#$?#.backlogo { background-image: none !important; }
 5donews.co.kr#$?#.body > .content.xe_content:not(:has(article)) { width: auto !important; }
-mule.co.kr#$?#.section-lefttop-center:has(> .mule-today) { width: 100% !important; }
\ No newline at end of file
+mule.co.kr#$?#.section-lefttop-center:has(> .mule-today) { width: 100% !important; }
```

Comment:
This rule must be located in `AnnoyancesFilter/sections/tweaks.txt` of AdguardFilters.
https://github.com/AdguardTeam/AdguardFilters/pull/87489

Reply:
deleted in 9a7fdd1